include/boost/stacktrace/detail/frame_msvc.ipp

```diff
@@ -18,7 +18,17 @@
 #include <boost/core/noncopyable.hpp>
 #include <boost/stacktrace/detail/to_dec_array.hpp>
 #include <boost/stacktrace/detail/to_hex_array.hpp>
+
+#ifdef WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#else
+// Prevent inclusion of extra Windows SDK headers which can cause conflict
+// with other code using Windows SDK
+#define WIN32_LEAN_AND_MEAN
```

Comment:
The downside of this solution is following:

If both conditions are met:

1. Boost.Stacktrace user's compilation unit includes windows.h header file after inclusion of Boost.Stacktrace header files.
1. Boost.Stacktrace user's compilation unit doesn't have WIN32_LEAN_AND_MEAN macro defined (IMHO, it is a rare case).
 
then the compilation unit loses "extra" Windows SDK headers (cderr.h, dde.h, ddeml.h, dlgs.h, lzexpand.h, mmsystem.h, nb30.h, shellapi.h, winperf.h, winsock.h, wincrypt.h, winefs.h, winscard.h), because windows.h header is not going to be included one more time into the same compilation unit, i.e. this solution potentially can cause regression for the users of Boost.Stacktrace, but that regression can be solved by including "extra" Windows SDK header files explicitly.